tornado-runtime/src/main/java/uk/ac/manchester/tornado/runtime/common/TornadoOptions.java

```diff
@@ -32,6 +32,21 @@ public class TornadoOptions {
     private static final String FALSE = "FALSE";
     private static final String TRUE = "TRUE";
 
+    /**
+     * Default OpenCL Compiler Flags.
+     */
+    public static final String DEFAULT_OPENCL_COMPILER_FLAGS = getProperty("tornado.opencl.compiler.flags", "-cl-mad-enable -cl-fast-relaxed-math -w");
+
+    /**
+     * Default PTX Compiler Flags.
+     */
+    public static final String DEFAULT_PTX_COMPILER_FLAGS = getProperty("tornado.ptx.compiler.flags", "");
```

Comment:
are we sure that PTX does not require any flags that we can use by default?

Reply:
I did not detect we pass any flags to the PTX driver compiler.